src/Range.jsx

```diff
@@ -62,7 +62,12 @@ class Range extends React.Component {
 
     const { bounds } = this.state;
     const value = nextProps.value || bounds;
-    const nextBounds = value.map((v, i) => this.trimAlignValue(v, i, nextProps));
+
+    if (value.length % 2 !== 0){
+      return
+    }
+
+    const nextBounds = value
```

Comment:
i assume this is doing something with not setting state if the incoming range has an odd number of items, but i'm not totally clear on the necessity of that

Reply:
sometimes, due to annoying logic mainly regarding the ability to join ranges, an array/range comes in with 1,3,5, etc, which gets filtered out by the above logic